sidecar/reloader/main.go

```diff
@@ -226,3 +219,12 @@ func requestWebhook(url url.URL, method string, bodys []byte, username, password
 	log.Println("status:", resp.Status)
 	log.Println("body:", string(body))
 }
+
+func isBlackFile(fileName string) bool {
```

Comment:
Maybe we can create `blackReloadList` in this function ? because there are not other function use `blackReloadList`, so `blackReloadList` doesn't need to be a global variable

Reply:
blackReloadList is a global configuration for the reloader

Reply:
Yes, but in fact it is not `configuration`, it is a `variable`, it can be modify, so I don't think this is a good idea

Reply:
Done